astronomer/providers/http/sensors/external_deployment_task.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Dict, Optional
+
+from astronomer.providers.http.sensors.http import HttpSensorAsync
+from astronomer.providers.http.triggers.http import ExternalDeploymentTaskTrigger
+from astronomer.providers.utils.typing_compat import Context
+
+
+class ExternalDeploymentTaskSensorAsync(HttpSensorAsync):
+    """
+    External deployment task sensor check for the response state success, continue polling till
```

Comment:
It should poll till the state is terminated i.e both success or fail instated of only success. WDYT?

Reply:
Fixed the doc string